ScoutSuite/providers/gcp/facade/utils.py

```diff
@@ -0,0 +1,19 @@
+import httplib2
+import googleapiclient
+from ScoutSuite.providers.utils import run_concurrently
+
+class GCPFacadeUtils:
+    @staticmethod
+    def _get_all(resources, resource_key, request, resources_group):
+        while request is not None:
+            response = request.execute()
+            resources.extend(response.get(resource_key, []))
+            request = resources_group.list_next(previous_request=request, previous_response=response)
+
+    @staticmethod
+    async def get_all(resource_key, request, resources_group):
+        resources = []
+        await run_concurrently(
+            lambda: GCPFacadeUtils._get_all(resources, resource_key, request, resources_group)
+        )
+        return resources
```

Comment:
I'm nitpicking, but I think it makes more sense to put `get_all` before `_get_all` (when you read the file, you start from a high level and you go deeper, it makes it more readable IMO).